index.html

```diff
@@ -0,0 +1,76 @@
+        integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF" crossorigin="anonymous">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Page Titles</title>
+    <script async src="script.js"></script>
+    <noscript>You need to enable JavaScript to view the full site.</noscript>
+</head>
+
+<body class="body">
+    <img class="topimage" src="https://i.ibb.co/Wzb3T0J/portfolio-top-image2.png" alt="portfolio-top-image2">
+    <div class="w-60 title">
+        <h1><em>Personal Statement</em></h1>
+        <p>I am currently a SMT Operator at ESI Electronics in Waterbury Connecticut. My current goal is to graduate
+            from
+            the University of Vermont's Coding Bootcamp <br>
+            and get a job as a web developer.</p>
+    </div> <br>
+    <div class="electronic-manufactoring">
+        <div class="w-50 work-experience">
+            <h2><u>Work Experience</u></h2>
+        </div>
+        <br>
+        <h2>Electronic Manufactoring</h2>
+
+        <div class="w-50 smt">
+            <img src="https://i.ibb.co/2WDP6P9/red-PCB.jpg" alt="printed circuit board" width="300px">
+            <p><strong>SMT Operator</strong></p>
+            <ul>
+                <li>Responsible for SMT feeder set-up, operation and minor adjustments</li>
+                <li>Repairing damaged or misplaced components on printed circuit boards</li>
+                <li>Performing virtual testing on all printed circuit boards to ensure proper operation</li>
+                <li>Keeping an open line of communication between SMT Operator, Engineers and Management</li>
+                <li>Repairing damaged or misplaced components on printed circuit boards</li>
+            </ul>
+        </div>
+        <br>
+        <div class="w-50 through-hole">
+            <img src="https://i.ibb.co/FBqSMcj/Green-PCB-Thru-Hole.jpg" alt="green printed circuit board" width="300px">
+            <p><strong>Through-Hole Technician</strong></p>
+            <ul>
+                <li>Install components according to assembly documentation</li>
+                <li>Touch-up/solder SMT and Through-hole components as necessary</li>
+                <li>Participate in continuous improvement and maintain a productive working relationship
+                    between SMT Operators and Engineers</li>
+                <li>Have the ability to assemble/solder parts and perform invenstory tasks as needed</li>
+                <li>Perform quality control on all completed PCBs</li>
+
+            </ul>
+
+        </div>
+    </div>
+    <div class="work-on">
+        <h3>Things I Need To Work On</h3>
+        <ul>
+            <li>Remembering key terms during class</li>
+            <li>Properly settings up my reposetories</li>
+            <li>Asking for help when i need it</li>
+            <li>Doing my own research after and before class</li>
```

Comment:
Rework or remove this section and change it to a list of things you are working on not a list of flaws.